portal-api/src/v1/api.js

```diff
@@ -287,7 +297,12 @@ const getUtilisationReports = async (bankId, reportPeriod) => {
       console.error('Get utilisation reports failed with the following report period: %s', reportPeriod);
       throw new Error('Invalid report period provided: %s', reportPeriod);
     }
-    const params = reportPeriod ? reportPeriod.start : undefined;
+
+    if (reportStatuses?.length > 0 && !reportStatuses.every(isValidReportStatus)) {
+      console.error('Get utilisation reports failed with the following report statuses: %s', reportStatuses);
+      throw new Error('Invalid report statuses provided: %s', reportStatuses);
+    }
+    const params = { reportPeriod: JSON.stringify(reportPeriod), reportStatuses };
```

Comment:
Bit tricky here - it appears that `axios` does not stringify your query params, but instead sends it as an object. When `express` then receives that `query`, it will parse it to an `object` rather than a JSON string representation of the object. This means that the `parseReportPeriod` helper in `dtfs-central-api` will always fail as the input will be an `object` which, when supplied to `JSON.parse`, is turned in the `string` `[object Object]` which is not valid JSON.

Tl;dr - I can just remove the helper and we don't have to worry about stringified JSON floating around